templates/branding/openSUSE/external_reporting.html.ep

```diff
@@ -35,6 +35,9 @@
 %         if ($subproduct eq 'Server' && $version eq '12') {
 %             $version = '12 (SLES 12)';
 %         }
+%         if ($subproduct eq 'Desktop-Updates' && $version eq '12 SP1') {
```

Comment:
I don't think the version is ever '12 SP1' (without dash). And the subproduct can be a lot, Server-Updates, Server-Incidents, ...